tests/test_all_cfgs.py

```diff
@@ -40,9 +32,24 @@ def get_test_cases():
         yield config_path
 
 
-@pytest.mark.pleasefixme
+@pytest.mark.skipif(
+    not os.getenv("TEST_DATA_ROOT"), reason="TEST_DATA_ROOT has to be defined for e2e config tests"
+)
 @pytest.mark.parametrize("config_path", get_test_cases())
 def test_configs(config_path, tmp_path):
+    TEST_DATA_ROOT = os.environ["TEST_DATA_ROOT"]
+    # we expect CONFIG_DIR and TEST_DATA_ROOT to have the same structure (e.g. <lang>/<dataset>)
```

Comment:
Correct name of `CONFIG_DIR` is currently `CONFIG_BASE_DIR`. Maybe we should rename it so it has the same style of naming as `TEST_DATA_ROOT`? e.g. name it `DATASET_CONFIGS_ROOT`?

Reply:
Thanks, fixed!